packages/web/tests/components/UserProfile/UserProfileLayout.test.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import {
+  UserProfile,
+  UserProfileProps,
+} from '../../../src/components/userprofile/UserProfile';
```

Comment:
This will help with coverage on the `index.ts` file

```suggestion
import { UserProfile } from '../../../src/components/userprofile';
import { UserProfileProps } from '../../../src/components/userprofile/UserProfile';
```